proposals/2023-03-upgrade-v2-0-6-ibc-go-upgrade/README.md

```diff
@@ -0,0 +1,39 @@
+# Panacea v2.0.6 Upgrade
+This is a proposal to do a software upgrade to the v2.0.6 software tag of the Panacea codebase on block height 9692000, which is estimated to occur on Wednesday March 08 2023 05:36:00 UTC 
```

Comment:
I think it would be good to use the same estimated time with [`upgrade.md`](https://github.com/medibloc/panacea-mainnet/blob/a58d1e60372bce839fde4f34aa2b671bbcc75cb1/panacea-3/v2.0.6/upgrade.md?plain=1#L15)

Reply:
fixed on committ [1880fce](https://github.com/medibloc/panacea-governance/pull/14/commits/1880fcee6277719fbf5f099e19d125ee826a6471)